integrations/rabbitmq/documentation.yaml

```diff
@@ -0,0 +1,95 @@
+exporter_type: included
+app_name_short: RabbitMQ
+app_name: {{app_name_short}}
+app_site_name: RabbitMQ
+app_site_url: https://rabbitmq.com/
+exporter_name: the RabbitMQ exporter
+exporter_pkg_name: rabbitmq
+exporter_repo_url: https://rabbitmq.com/prometheus.html#overview-prometheus
+minimum_exporter_version: v3.8.0
+additional_prereq_info: |
+  {{app_name_short}} exposes Prometheus-format metrics automatically; you do not
+  have to install it separately. To verify that {{exporter_name}} is emitting metrics on the expected
```

Comment:
This is a little misleading: To verify that ...., set up port-forwarding with the following command". Setting up port forwarding doesn't verify that the metrics are emitted -- it's part of it, but not the whole thing.

The line  "To determine if ..., run the following command" is similarly confusing -- that just sets up port forwarding.

What about something like this:

To verify that {{exporter_name}} is emitting metrics on the expected endpoints, do the following:

0. Set up port forwarding by using the following command: (lines 17-19)
0. Access the endpoint \`localhost:15692/metrics\` by using the browser or the \`curl\` utility in another terminal session.